client/components/payments-activity/style.scss

```diff
@@ -10,45 +26,121 @@
 	}
 }
 
-.payments-activity-data {
-	display: flex;
-	padding: 24px;
+.wcpay-payments-activity-data {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	width: 100%;
 
 	@include breakpoint( '<660px' ) {
-		flex-direction: column;
+		grid-template-columns: 1fr;
+		padding: 24px;
 	}
 
-	.total-payments-volume {
-		display: flex;
-		flex: 2;
-		flex-direction: column;
-		align-items: flex-start;
+	.wcpay-payments-data-highlights {
+		display: grid;
+		grid-template-columns: 1fr 1fr;
 
-		&__heading {
-			font-size: 16px;
-			font-weight: 600;
-			line-height: 16px;
-			color: $gray-900;
+		@include breakpoint( '<660px' ) {
+			grid-template-columns: 1fr;
 		}
 
-		&__body {
-			font-size: 24px;
-			font-weight: 600;
-			color: $gray-900;
-			line-height: 32px;
-			margin-top: 14px;
+		&__item {
+			padding: 24px;
+			border-left: 1px solid $gray-200;
+			display: flex;
+			flex-direction: column;
+			justify-content: flex-start;
+			align-items: flex-start;
+			min-height: 129px;
+
+			&:hover {
+				.wcpay-payments-data-highlights__item__wrapper__report-link {
+					visibility: visible;
+				}
+			}
+
+			&__label {
+				@include label-styles;
+				margin: 0 0 8px 0;
+			}
+
+			&__wrapper {
+				display: flex;
+				flex-direction: column;
+				align-items: flex-start;
+				width: 100%;
+
+				@include breakpoint( '<660px' ) {
+					flex-direction: row;
+					justify-content: space-between;
+					align-items: center;
+				}
+
+				&__amount {
+					@include amount-styles;
+					margin: 0 0 8px 0;
+
+					@include breakpoint( '<660px' ) {
+						margin: 0;
+					}
+				}
+
+				&__report-link {
+					text-decoration: none;
+					font-size: 12px;
+					visibility: hidden;
+
+					@include breakpoint( '<660px' ) {
+						min-height: unset;
+						visibility: visible;
+					}
+				}
+			}
+
+			@include breakpoint( '<660px' ) {
+				border-left: none;
+				border-bottom: 1px solid $gray-200;
+				padding: 16px 0;
+				min-height: unset;
+			}
 		}
 
-		.gridicon {
-			height: 16px;
-			width: 16px;
-			margin-left: 4px;
-			vertical-align: text-bottom;
+		@include breakpoint( '<660px' ) {
+			flex-direction: column;
 		}
 	}
 
-	.payments-data-highlights {
-		display: flex;
-		flex: 3;
+	#wcpay-payments-data-highlights {
+		&__charges,
+		&__refunds {
+			border-bottom: 1px solid $gray-200;
+		}
```

Comment:
Rather than target IDs specific to the content of a tile, consider using generic selectors which makes things easier if the content ever changes (e.g. order of tiles change).

E.g. we could use `nth-of-type()` up where

```scss
.wcpay-payments-data-highlights__item {
  &:nth-of-type( 3 ),
  &:nth-of-type( 4 ) {
    border-top: 1px solid $gray-200;
  }
```

Reply:
Nice, ideally this styling is managed by the container rather than the children (it knows where they are needed). Another alternative would be a class/prop so author can explicitly enable the spacer line where needed.

Reply:
This is a great suggestion! Thanks! Implemented here - https://github.com/Automattic/woocommerce-payments/pull/8564/commits/d44bac03f85f7e66a5a400309000b5428d9ba4ba